ci-operator/step-registry/install-operators/install-operators-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: install-operators
+  steps:
+  - ref: install-operators
+  - ref: cucushift-installer-check-operators
```

Comment:
Good find! I never knew a check-operators step existed.

I referred to the [cucushift-installer-check-cluster-health](https://steps.ci.openshift.org/reference/cucushift-installer-check-cluster-health) step in the last PR as a possible good step to use.

Let's bring this topic to the parking lot tomorrow as I think if we use these it should be consistent across all of our jobs and I'd like to get the teams feedback on that.

Reply:
> Good find! I never knew a check-operators step existed.
> 
> I referred to the [cucushift-installer-check-cluster-health](https://steps.ci.openshift.org/reference/cucushift-installer-check-cluster-health) step in the last PR as a possible good step to use.
> 
> Let's bring this topic to the parking lot tomorrow like you mentioned in the [slack thread](https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1700062766601229?thread_ts=1699824507.568839&cid=C04PK4QPSR1) if we use these it should be consistent across all of our jobs and I'd like to get the teams feedback on that.

Moved this implementation to #45773, will be configured after team discussions.